packages/lab/src/vertical-navigation/VerticalNavigationItemTrigger.tsx

```diff
@@ -0,0 +1,46 @@
+import { makePrefixer, type RenderPropsType, renderProps } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import { clsx } from "clsx";
+import { type ComponentPropsWithoutRef, forwardRef } from "react";
+import verticalNavigationItemTriggerCss from "./VerticalNavigationItemTrigger.css";
+
+// biome-ignore lint/suspicious/noExplicitAny: <explanation>
+function ItemAction(props: any) {
+  return renderProps("a", props);
+}
+
+export interface VerticalNavigationItemTriggerProps
+  extends ComponentPropsWithoutRef<"a"> {
+  render?: RenderPropsType["render"];
+}
+
+const withBaseName = makePrefixer("saltVerticalNavigationItemTrigger");
+
+export const VerticalNavigationItemTrigger = forwardRef<
```

Comment:
We have a few Trigger components now.
Some are actions, some use children as their actions. (do we need to define what a trigger is ?)

This one seems to exist to wrap an action in styles, so could it just be that and define the action through it's children ?